### ISSUE TITLE
[19:08:31] [ERROR] [LocalAudioTrackExecutor]: Error in playback of KJmcEAwazD0

### ISSUE THREAD

**Issue by Truzedo** [jagrosh/MusicBot#452]
[debug_information.txt](https://github.com/jagrosh/MusicBot/files/4867074/debug_information.txt)
**Describe the bug**
Youtube changed their API calls again I know, so this may be a side effect.  But anytime we start the bot up with our standard playlist we keep getting several songs skipped at beginning of default list.  This started happening about 2 versions ago.  But happens consistently in the last 2 weeks from today.

**To Reproduce**
1. Run with '...'
2. Do a command '....'
3. Some other step '....'

**Expected behavior**
Default playlist play as normal and not have any "Something broke when playing the track" messages in nohup.out

**Screenshots**
beatbot@mcmain:~$ view nohup.out
[19:08:28] [INFO] [Startup]: Loaded config from /home/beatbot/config.txt
[19:08:28] [INFO] [JDA]: Login Successful!
[19:08:29] [INFO] [WebSocketClient]: Connected to WebSocket
[19:08:30] [INFO] [JDA]: Finished Loading!
[19:08:31] [ERROR] [LocalAudioTrackExecutor]: Error in playback of KJmcEAwazD0
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:117)
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: java.io.IOException: Invalid status code for video page response: 429
        at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.loadDetails(DefaultYoutubeTrackDetailsLoader.java:28)
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:75)
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:42)
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:105)
        ... 4 common frames omitted
Caused by: java.io.IOException: Invalid status code for video page response: 429
        at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.load(DefaultYoutubeTrackDetailsLoader.java:39)
        at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.loadDetails(DefaultYoutubeTrackDetailsLoader.java:26)
        ... 7 common frames omitted
[19:08:31] [ERROR] [LocalAudioTrackExecutor]: Error in playback of gYR0xP1j4PY
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:117)

**Debug Info**
JMusicBot-0.2.9-Linux.jar
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.4 LTS
Release:        18.04
Codename:       bionic
Java Version
java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
Please run the `debug` command and paste it here or upload it as an attachment. If you cannot run the command, please provide information about your system such as operating system version, JMusicBot version, etc.

**Additional context**
Discord channel does not reflect current song its playing anymore either.

**Comment by Sanduhr32** [comment]
Caused by: java.lang.RuntimeException: java.io.IOException: Invalid status code for video page response: 429

You are being blocked by YouTube, because you use it to frequently, without being signed in / solving anti spam captchas.

**Comment by jagrosh** [comment]
Duplicate of #305
